### PR TITLE
Migrate inline javascript event handlers

### LINKED ISSUE #860: [Panorama Partners Development] Migrate targetedms inline event handlers
**Tasks**
- [x] PlotTypeCheckCombo.js
- [x] protein.js
- [x] calibrationCurve.jsp  (2 onchange handlers)
- [x] chromatogramsForm.jsp  (4 onclick handlers)
- [x] figuresOfMerit.jsp  (1 onclick handler)
- [x] pharmacokinetics.jsp  (2 onclick handlers)
- [x] Test fixes
- [x] Merge

### PR DESCRIPTION
**Pull request by ankurjuneja**
#### Rationale
A strict (no 'unsafe-inline' directive) CSP forbids all inline events. Instead, events must be attached via JavaScript inside a properly nonced script tag

#### Changes
*  PlotTypeCheckCombo.js
* protein.js
* calibrationCurve.jsp (2 onchange handlers)
* chromatogramsForm.jsp (4 onclick handlers)
* figuresOfMerit.jsp (1 onclick handler)
* pharmacokinetics.jsp (2 onclick handlers)

### CODE REVIEW

**Review by labkey-jeckels — CHANGES_REQUESTED** [review]
org.labkey.test.tests.targetedms.TargetedMSCalibrationCurveTest.testCalibrationPK appears to be failing in the branch but passing in develop

**Comment by ankurjuneja** [comment]
the TC is clear now

File: src/org/labkey/targetedms/view/pharmacokinetics.jsp [hunk 1]
```diff
@@ -52,7 +52,7 @@
     %>
     <labkey:panel title="<%=subgroupTitle%>" type="portal">
         <div id="targetedms-fom-export" class="export-icon" data-toggle="tooltip" title="Export to Excel">
-            <i class="fa fa-file-excel-o" onclick="exportExcel('<%=h(subgroup)%>')"></i>
+            <%=iconLink("fa fa-file-excel-o", "", null).onClick("exportExcel('" + h(subgroup) + "')") %>
```
Comment (labkey-jeckels) [inline src/org/labkey/targetedms/view/pharmacokinetics.jsp]: I suspect we don't need the `h()` call here. I hope `onClick()` already does the appropriate encoding for us.

File: src/org/labkey/targetedms/view/pharmacokinetics.jsp [hunk 2]
```diff
@@ -82,7 +82,8 @@
                 <span id="nonIVC0Controls-Warn-<%=h(subgroup)%>" class="labkey-error"><h4>WARNING: Please enter a non-IV C0 and recalculate.</h4></span>
                 non-IV C0
                 <input type="number" id="nonIVC0-<%=h(subgroup)%>" label="non-IV C0"/>
-                <button id="btnNonIVC0-<%=h(subgroup)%>" onclick="updateStatsForNonIVC0('<%=h(subgroup)%>')">Recalculate</button>
+                <% addHandler("btnNonIVC0-"+h(subgroup), "click", "updateStatsForNonIVC0('"+ h(subgroup) + "')"); %>
```
Comment (labkey-jeckels) [inline src/org/labkey/targetedms/view/pharmacokinetics.jsp]: Similar here - probably don't need the call to `h()` or we'll get double-encoding

**Review by labkey-jeckels — APPROVED** [review]

**Comment by labkey-adam** [comment]
@ankurjuneja looks like the `figuresOfMerit.jsp` change didn't make it into this PR

**Pull request merged**